### PR TITLE
lists table in alphabetical order

### PR DESCRIPTION
**Pull request by karthequian**
Fixed a couple of things as well: 

1. `Oracle Cloud Infrastructure Container Engine for Kubernetes (OKE)` was `IaaS Provider Oracle Cloud Infrastructure Container Engine for Kubernetes (OKE)` 
2. capitalized all instances of GCE

### CODE REVIEW

**Comment by karthequian** [comment]
Originally from a conversation with @cody-clark in https://github.com/kubernetes/website/pull/13137#issuecomment-472152331

**Comment by cody-clark** [comment]
This looks great, @karthequian! Thanks for taking the time to reorganize this. I have just one request:

File: content/en/docs/setup/pick-right-solution.md [hunk 1]
```diff
@@ -233,57 +233,57 @@ Below is a table of all of the solutions listed above.
...
-Agile Stacks       | Terraform   | CoreOS | multi-support | [docs](https://www.agilestacks.com/products/kubernetes) | Commercial
-IBM Cloud Kubernetes Service | | Ubuntu | calico | [docs](https://cloud.ibm.com/docs/containers?topic=containers-container_index#container_index) | Commercial
-Digital Rebar        | kubeadm      | any    | metal       | [docs](/docs/setup/on-premises-metal/krib/)                                  | Community ([@digitalrebar](https://github.com/digitalrebar))
 VMware Cloud PKS     |              | Photon OS | Canal | [docs](https://docs.vmware.com/en/VMware-Kubernetes-Engine/index.html) | Commercial
 VMware Enterprise PKS     | BOSH       | Ubuntu | VMware NSX-T/flannel | [docs](https://docs.vmware.com/en/VMware-Enterprise-PKS/) | Commercial
-Mirantis Cloud Platform | Salt | Ubuntu | multi-support | [docs](https://docs.mirantis.com/mcp/) | Commercial
-IAAS Provider- Oracle Cloud Infrastructure Container Engine for Kubernetes (OKE) |              |              | multi-support | [docs](https://docs.cloud.oracle.com/iaas/Content/ContEng/Concepts/contengoverview.htm) | Commercial
 
 {{< note >}}
```
Comment (cody-clark) [inline content/en/docs/setup/pick-right-solution.md]: can you delete this note? Thanks again!

**Comment by cody-clark** [comment]
Preview: https://deploy-preview-13143--kubernetes-io-master-staging.netlify.com/docs/setup/pick-right-solution/#table-of-solutions

File: content/en/docs/setup/pick-right-solution.md (hunk 1 above)
Comment (karthequian) [reply content/en/docs/setup/pick-right-solution.md]: Ofcourse!

**Comment by karthequian** [comment]
@cody-clark this should be good to go! Let me know if you need anything else on it.

**Comment by DanyC97** [comment]
@karthequian could you please rebase ? thx

**Comment by karthequian** [comment]
Totally not what I expected to happen when I rebased. Going to make a new PR for this instead.

**Comment by karthequian** [comment]
@DanyC97 sorry I took a while to get to this- looks like the merge is a bit more complicated, so I just created a new branch and did it here instead: https://github.com/kubernetes/website/pull/13267. Closing this PR for the new one.

**Pull request closed without merging**